src/condor_ce_jobmetrics

```diff
@@ -123,15 +157,19 @@ def main():
                 print >> sys.stderr, "Schedd", ad['Name'], "VO", vo, "totals:", info
             rrdtool.update(vo_fname, "N:%d:%d:%d:%d" % (info['Running'], info['Idle'], info['Held'], info['Jobs']))
 
-    job_results_final = []
-    for key, val in job_results.items():
-        val['DN'] = key[0]
-        val['VO'] = key[1]
-        val['VOMS'] = key[2]
-        job_results_final.append(val)
+    job_results_final = list_job_results_final(job_results)
     pilots_fname = rrd.path_with_spool(environ, "pilots")
     secure_json_write(pilots_fname, job_results_final)
 
+    gpu_job_results_final = list_job_results_final(gpu_job_results)
+    gpu_pilots_fname = rrd.path_with_spool(environ, "gpu_pilots")
```

Comment:
```suggestion
    # gpu_job_results_final = list_job_results_final(gpu_job_results)
    # gpu_pilots_fname = rrd.path_with_spool(environ, "gpu_pilots")
```

Since we're not writing out the rrd, we don't really need to bother with these calls. Keeping them commented out seems fine in case we want them in the future but it'd be nice to include a comment explaining that.